K2Bridge.Tests.UnitTests/JsonConverters/AggregationConverterTests.cs

```diff
@@ -73,143 +73,150 @@ public class AggregationConverterTests
             }}";
 
         private const string NoAggAggregation = @"
-        {""aggs"": { 
-            ""2"": {
-                ""noagg"" : { 
-                    ""field"" : ""metric"" 
-                    } 
-            }
-        }}";
+            {""aggs"": { 
+                ""2"": {
+                    ""noagg"" : { 
+                        ""field"" : ""metric"" 
+                        } 
+                }
+            }}";
 
-        private static readonly Aggregation ExpectedValidDateHistogramAggregation = new Aggregation()
+        private static readonly AggregationContainer ExpectedValidDateHistogramAggregation = new AggregationContainer()
         {
             PrimaryAggregation = null,
-            SubAggregations = new Dictionary<string, Aggregation>
+            SubAggregations = new AggregationDictionary
+            {
+                ["2"] = new AggregationContainer
                 {
-                    { "2", new Aggregation() {
-                        PrimaryAggregation = new DateHistogramAggregation {
-                            FieldName = "timestamp",
-                            FieldAlias = "_2",
-                            Interval = "1m",
-                            Metric = "count()",
-                            },
-                        SubAggregations = new Dictionary<string, Aggregation>(),
-                        }
+                    PrimaryAggregation = new DateHistogramAggregation
+                    {
+                        Field = "timestamp",
+                        Key = "2",
+                        FixedInterval = "1m",
+                        TimeZone = "Asia/Jerusalem",
+                        MinimumDocumentCount = 1,
                     },
+                    SubAggregations = new AggregationDictionary(),
                 },
+            },
         };
 
-        private static readonly Aggregation ExpectedValidTermsAggregation = new Aggregation()
+        private static readonly AggregationContainer ExpectedValidTermsAggregation = new AggregationContainer()
         {
             PrimaryAggregation = null,
-            SubAggregations = new Dictionary<string, Aggregation>
+            SubAggregations = new AggregationDictionary
             {
+                ["2"] = new AggregationContainer
                 {
-                    "2", new Aggregation() {
-                        PrimaryAggregation = new TermsAggregation {
-                            FieldName = "DestCountry",
-                            FieldAlias = "_2",
-                            Metric = "count()",
-                            SortFieldName = "_count",
-                            SortOrder = "desc",
+                    PrimaryAggregation = new TermsAggregation
+                    {
+                        Field = "DestCountry",
+                        Key = "2",
```

Comment:
IMO, alias was more self explanatory

Reply:
Yes. I was hesitating with this change. When reading the code, I realized this property indicates the associated dictionary key where aggregation is located. So I updated the name with this logic. But Alias is also fine for me and I can rollback this change.  @tomconte @nebrass -> Your opinion ?

Reply:
Personally I think Key is more relevant than alias

Reply:
+1 for Key, since this represents the key that identifies the aggregation; the value is used as a field alias in the KustoQL query, but I think from the request/response model perspective, Key makes sense.

Reply:
Key is approved 🥇